### PR TITLE
Finished: Setup-math magician: 

### PR DESCRIPTION
**Pull request by Zafron047**
Dear reviewer,
Please see the below changes made and review this PR. Please advice additional changes if required.

**Fulfilled below requirements:**
- [x] Made sure that there are no linter errors
- [x] Made sure that you used correct Gitflow.
- [x] Made sure that you documented your work in a professional way
- [x] Made sure I have Node.js installed on your machine
- [x] Created a React application is using Create React App
- [x] The application run in the browser without errors

Best,

Zafron. 😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+
+- Thanks to all Microverse community
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (gemmen29) [inline README.md:151]: - [ ] Kindly fix the MIT license link in your README because when I press it I get a 404 not found error.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Zafron047, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Zafron047) [reply README.md:151]: @gemmen29 Thanks for the thorough check.
I have added a mit license now. 😃

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Zafron047, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zafron047** [comment]
@ShadyShawkat Thanks a lot for the review and approval. 😄

**Pull request merged**